### ISSUE TITLE
Upgrade kubernetes 1.23 version

### ISSUE THREAD

**Issue by ronin201** [aws/eks-anywhere#1610]
Is upgrade available for 1.23 kubernetes cliuster version?

I received the error:
kubernetes version 1.23 is not supported by bundles manifest 4

**Comment by TerryHowe** [comment]
That is correct. EKS tends to favor proven releases over cutting edge and the release schedule is always behind the k8s release cycle. EKS also supports older releases of k8s longer than upstream. I'm not sure what the schedule is for 1.23.

**Comment by jaxesn** [comment]
We are cutting a release later this week which will have 1.22 support.  The 1.23 schedule is not set at this point.

**Comment by ronin201** [comment]
so is 1.22 supported for update?

**Comment by ronin201** [comment]
Hello any news regarding 1.22 support in upgrade manifest?

**Comment by ronin201** [comment]
eksctl anywhere upgrade cluster -f dev-cluster.yaml

result:

Error: failed to upgrade cluster: unable to get cluster config from file: kubernetes version 1.22 is not supported by bundles manifest 4
